src/Makefile

```diff
@@ -23,7 +23,7 @@
 
 TARGET := dpvs
 
-VERSION_STRING := $(shell ./VERSION)
+VERSION_STRING := $(shell git describe --tags --always)
```

Comment:
It's good idea that using command get version automatically, however some scripts may need VERSION file to parse the DPVS version on deployed machine, which has no source code installed.
We can change that scripts to use 'dpvs --version', but for now, we need compatible with these scripts.

so 
1. pls check VERSION and use file if exist, if not exist use git command.
2. reserve VERSION file for compatibility.

Reply:
Done and re-pulled the rqeuest.